deepmd/tf/infer/deep_eval.py

```
@@ -1197,7 +1197,7 @@ def __init__(
         self.neighbor_list = neighbor_list
 
     @property
-    @lru_cache(maxsize=None)
+    @cache
```

Ruff

Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)